modules/dnn/src/layers/recurrent_layers.cpp

```diff
@@ -127,6 +127,8 @@ class LSTMLayerImpl CV_FINAL : public LSTMLayer
     bool useAVX2;
 #endif
 
+    std::vector<Mat> cudaWorkaround;
```

Comment:
It may be also not a workaround, since it looks like not a bug, but a special memory arrangement. May be `blobsCudaLayout` or `blobsForCudaInference` will suit better.